src/db.js

```
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '112478',
```

Security/style: currently the code falls back to a hard-coded default password `'112478'`. Avoid embedding default secrets in source. Consider using `POSTGRES_PASSWORD || undefined` (or fail with a clear error message if password is required) so you don't commit a password in code. Also convert the port to a number with `Number(POSTGRES_PORT) || 5432` to ensure Sequelize receives a numeric port.